packages/wdio-runner/src/index.js

```diff
@@ -60,6 +60,8 @@ export default class Runner extends EventEmitter {
         this.configParser.filterWorkerServices()
 
         this.config = this.configParser.getConfig()
+        this.config.sharedStoreServicePort = sharedStoreServicePort
```

Comment:
where is this used?

Reply:
It's used in the service part of the service. I want to run only one instance of server on a random port to avoid conflicts but it's not trivial to pass data (port) from main process to child processes. Anyway I'd need to send an event with payload to child process or something like that.

Reply:
anyway I've removed it, added it temporary because it was easier (for me) to pass data this way.